web/themes/joinup/templates/join-collection-form.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * @file
+ * Template to render the share content form.
```

Comment:
I quickly checked the [docs](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21theme.api.php/function/hook_theme/8.2.x) and some examples in core, with a bit of specificity, I think it will be possible to move the templates in the `form` folder. I quickly checked with this, please double-check if it also works for you? If yes, could you please move them in the form folder and apply this suggestion?
[forms_path.txt](https://github.com/ec-europa/joinup-dev/files/1190841/forms_path.txt)